client/influxdb.go

```
@@ -186,9 +187,9 @@ func (c *Client) Query(q Query) (*Response, error) {
 // QueryContext sends a command to the server and returns the Response
 // It uses a context that can be cancelled by the command line client
 func (c *Client) QueryContext(ctx context.Context, q Query) (*Response, error) {
-	u := c.url
+	u, _ := url.Parse(c.url.String())
```

Please don't ignore any errors from `url.Parse`. Return them to the caller